docs/tutorials/essentials/part-1-overview-concepts.md

```diff
@@ -397,7 +397,7 @@ console.log(store.getState())
 
 #### Selectors
 
-**Selectors** are functions that know how to extract specific pieces of information from a store state value. As an application grows bigger, this can help avoid repeating logic as different parts of the app need to read the same data:
+**Selectors** are functions that know how to extract specific pieces of information from a store state value. As an application grows bigger, this can help avoid repeating logic since different parts of the app need to read the same data.
```

Comment:
```suggestion
**Selectors** are functions that know how to extract specific pieces of information from a store state value. As an application grows bigger, this can help avoid repeating logic since different parts of the app need to read the same data:
```